cblox/include/cblox/core/tsdf_esdf_submap.h

```diff
@@ -31,9 +39,11 @@ class TsdfEsdfSubmap : public TsdfSubmap {
   // Generate the ESDF from the TSDF
   void generateEsdf();
 
+  void setTsdfMap(const voxblox::Layer<TsdfVoxel>& tsdf_layer);
```

Comment:
I suspect that this will _copy_ a layer into this submap. I'm not sure the circumstances under which one might want to do that. At the moment the design intent is that each submap is the sole owner of a single layer/TsdfMap.